src/sentry/shared_integrations/client/base.py

```diff
@@ -450,6 +451,9 @@ def disable_integration(self) -> None:
             integration_service.update_integration(
                 integration_id=rpc_integration.id, status=ObjectStatus.DISABLED
             )
+            from sentry.integrations.notifydisable import notify_disable
```

Comment:
Can you import this at the top of the file too?

Reply:
Generally you always want to import at the top of the file unless there's a circular dependency that requires it being imported inside a function.